modules/Makefile.kubernetes

```diff
@@ -112,10 +133,15 @@ kubernetes\:deploy-service:
 ## Deploy controller and service; notify datadog
 kubernetes\:deploy:
 	$(NOTIFY_STARTING)
+	@[ ! -f "$(KUBERNETES_RESOURCE_PATH)/$(KUBERNETES_APP)-deployment.yml" ] || $(SELF) kubernetes:deploy-deployment || $(NOTIFY_FAILURE)
```

Comment:
Both resource types are supported. E.g. a `DaemonSet` is a type of controller, but a `Deployment` resource type creates `ReplicationController` resources. A `Deployment` type can only create `ReplicationController` resources and nothing else.